Sources/Signal.swift

```diff
@@ -1404,15 +1388,17 @@ extension Signal {
 	}
 
 
-	/// Send only the final value and then immediately completes.
+	/// Accumuate all values from `self`, and forward only the final accumuated result.
```

Comment:
I don't think `accumulate` (typoed) is the best description. The return value is the accumulator, but I don't think _accumulate_ describes the operation itself.

C.f. the Swift `reduce` description:

> Returns the result of combining the elements of the sequence using the given closure